src/main/scala/com/lunatech/pointingpoker/actors/Room.scala

```diff
@@ -0,0 +1,116 @@
+package com.lunatech.pointingpoker.actors
+
+import java.util.UUID
+
+import akka.actor.typed.{ActorRef, Behavior}
+import akka.actor.typed.scaladsl.{ActorContext, Behaviors}
+import akka.actor.{ActorRef => UntypedRef}
+import com.lunatech.pointingpoker.websocket.WSMessage
+import com.lunatech.pointingpoker.websocket.WSMessage.MessageType
+
+object Room {
+
+  sealed trait Command
+  final case class Join(user: User)                                     extends Command
+  final case class Leave(userId: UUID)                                  extends Command
+  final case class Vote(userId: UUID, estimation: String)               extends Command
+  final case class ClearVotes(userId: UUID)                             extends Command
+  final case class ShowVotes(userId: UUID)                              extends Command
+  final case class EditIssue(userId: UUID, issue: String)               extends Command
+  private[actors] final case class GetData(replyTo: ActorRef[Response]) extends Command
+
+  sealed trait Response
+  final case class DataStatus(data: RoomData) extends Response
```

Comment:
As far as I can see, this is only used as a response to `GetData` and only ever returns `RoomData` so it is not really needed. `GetData` above can be changed to be just:
```
   private[actors] final case class GetData(replyTo: ActorRef[RoomData]) extends Command
```

Reply:
I would like to keep the structure for more responses in case the DSL grows.